src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -60,6 +66,14 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
   );
 };
 
+ItemUser.propTypes = {
+  photo: PropTypes.string,
+  username: PropTypes.string,
+  followed: PropTypes.any,
+  onPress: PropTypes.func,
+  userid: PropTypes.string,
+  isAnon: PropTypes.string
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding PropTypes is a great practice for type checking in React. However, the `isAnon` prop is declared as a string. If this prop is intended to be a boolean indicating whether the user is anonymous or not, it should be checked as a boolean.
```diff
-  isAnon: PropTypes.string
+  isAnon: PropTypes.bool
```
This will help prevent bugs related to incorrect data types.


<!-- This is an auto-generated comment by OSS CodeRabbit -->